smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -574,4 +717,23 @@ contract BridgeBank is BankStorage,
 
         emit LogUnlock(recipient, token, amount);
     }
+
+    function setBridgeTokenDenom(
+      address _token, string memory _denom
+    ) public onlyOperator returns (bool) {
```

Comment:
There should just be a function called ```forceSetBridgeTokenDenom``` which anyone can call that automatically sets things, and another separate function that does what this function does.

This function should change to onlyOwner.